server/startup.coffee

```diff
@@ -1,11 +1,16 @@
 incidentReportSchema = require '/imports/schemas/incidentReport.coffee'
 Incidents = require '/imports/collections/incidentReports.coffee'
+Articles = require '/imports/collections/articles.coffee'
 
 Meteor.startup ->
   # Clean-up curatorInboxSourceId when user goes offline
   Meteor.users.find({'status.online': true}).observe
     removed: (user) ->
       Meteor.users.update(user._id, {$set : {'status.curatorInboxSourceId': null}})
+  # update articles to use arrays instead of strings for their UserEventId values
```

Comment:
I changed the way database updates work. This should go at the end of updaters.coffee and the data version needs to be incremented.